website/content/docs/networking/cni.mdx

```diff
@@ -1,93 +1,62 @@
-  configurations.
+@include 'install/bridge-iptables.mdx'
 
-* `.conf` and `.json` files are loaded as individual [plugin
-  configurations][cni_spec_plugin_config] for a specific network.
+### Verify cgroup controllers
 
-## Using CNI networks with Nomad jobs
+@include 'install/cgroup-controllers.mdx'
 
-To specify that a job should use a CNI network, set the task group's
-network [`mode`][] attribute to the value `cni/<your_cni_config_name>`.
-Nomad will then schedule the workload on client nodes that have fingerprinted a
-CNI configuration with the given name. For example, to use the configuration
-named `mynet`, you should set the task group's network mode to `cni/mynet`.
-Nodes that have a network configuration defining a network named `mynet` in
-their [`cni_config_dir`][] will be eligible to run the workload.
+## Create bridge mode configuration
 
-## Nomad's `bridge` configuration
+Nomad itself uses CNI plugins and configuration as the underlying implementation for the `bridge` network mode, using the [loopback](https://github.com/containernetworking/plugins#main-interface-creating), [bridge](https://www.cni.dev/plugins/current/main/bridge/), [firewall](https://www.cni.dev/plugins/current/meta/firewall/), and [portmap](https://www.cni.dev/plugins/current/meta/portmap/) CNI plugins configured together to create Nomad's bridge network.
 
-Nomad itself uses CNI plugins and configuration as the underlying implementation
-for the `bridge` network mode, using the [loopback][], [bridge][], [firewall][],
-and [portmap][] CNI plugins configured together to create Nomad's bridge
-network.
+Nomad uses the following configuration template when setting up a bridge network. The template placeholders have been replaced with the default configuration values for [`bridge_network_name`](/nomad/docs/configuration/client#bridge_network_name), [`bridge_network_subnet`]( /nomad/docs/configuration/client#bridge_network_subnet), and an internal constant that provides the value for `iptablesAdminChainName`. You can use this template as a basis for your own CNI-based bridge network configuration in cases where you need access to an unsupported option in the default configuration, like hairpin mode. When making your own bridge network based on this template, ensure that you change the `iptablesAdminChainName` to a unique value for your configuration.
 
-The following is the configuration template Nomad uses when setting up these
-networks with the template placeholders replaced with the default configuration
-values for [`bridge_network_name`][], [`bridge_network_subnet`][], and an
-internal constant that provides the value for `iptablesAdminChainName`. This is
-a convenient jumping off point to discuss a worked example.
+Refer to the [CNI Specification](https://github.com/containernetworking/cni/blob/main/SPEC.md) for more information on this configuration format.
+
+Create your configuration in the `/opt/cni/config` directory.
```

Comment:
This is correct, by default, but may conflict with the later section https://nomad-ls5gkbami-hashicorp.vercel.app/nomad/docs/networking/cni#configure-nomad-clients where we describe that this directory (and the `/opt/cni/bin` directory can be configured. Do we want to add a reference to that section here, or is the goal to just show the defaults and let the user figure out from context they can set it to non-default later? (Which seems reasonable, if so)

Reply:
Added to partial that has the install CNI reference content